packages/viz/src/lib/sources/CARTOSource.ts

```diff
@@ -149,7 +149,8 @@ export class CARTOSource extends Source {
 
     const dimensions: Record<string, { column: string }> = {};
     fields.forEach(field => {
-      if (field.aggregation) {
+      // prevent using cartodb_id as aggregation column
+      if (field.aggregation && field.column !== 'cartodb_id') {
```

Comment:
how about a DEFAULT_ID_PROPERTY const?

For similar topics in the future we can check maps instantiation params in VL, like in_:
https://github.com/CartoDB/carto-vl/blob/57a55888a67d8e76bca8a259afaa992722a95c55/src/client/windshaft.js